R/internals.R

```diff
@@ -379,3 +379,20 @@
 #   if(getOption("mypackage.verbose", default = TRUE)) message(glue::glue(msg))
 #   return(NULL)
 # }
+
+#' Load Page with headers
+#'
+#' loads webpages with a header passed to read_html
+#'
+#' @param page_url url of the page wanted to be loaded
+#'
+#' @return a html webpage
+#'
+#' @noRd
+#'
+.load_page <- function(page_url) {
+  agent <- getOption("worldfootballR.agent", default = "RStudio Desktop (2022.7.1.554); R (4.1.1 x86_64-w64-mingw32 x86_64 mingw32)")
```

Comment:
i don't think the `default` will ever actually be used, since `.onLoad` in `zzz.R` should always be called implicitly the first time someone uses a function from the package in a given session. anyways, in case i'm wrong, there's no harm in defining a default that we know will work.